app/controllers/api/v1/web_hooks_controller.rb

```diff
@@ -30,14 +30,24 @@ def remove
   end
 
   def fire
-    webhook = current_user.web_hooks.new(url: @url)
     @rubygem = Rubygem.find_by_name("gemcutter") unless @rubygem
```

Comment:
since you are touching this method... does rubygem need to be an instance variable? looks like it doesn't